pdc/apps/release/tests.py

```diff
@@ -1047,7 +1072,7 @@ def setUp(self):
     def test_update(self):
         response = self.client.put(self.url,
                                    {'short': 'product', 'version': '1.0', 'release_type': 'ga',
-                                    'name': 'Our Product'},
+                                    'name': 'Our Product', 'sigkey': 'ABCDEF'},
```

Comment:
Maybe we can have test method test_update_sigkey separately which will verify sigkey update behavior

Reply:
Make sense, add a new function here. please review.